src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -33,37 +35,54 @@ export type DownloadConfirmationState = {
   fileCount: number
   fileSize: number
   downloadEstimate?: string
-  status: StatusEnum
   errorMessage?: string
-  ownerId?: string
 }
+
+type QueryOrFolder = {
+  queryBundleRequest?:QueryBundleRequest
+  folderId?: string
+}
+
 export type DownloadConfirmationProps = {
-  fnClose?: Function
+  fnClose?: () => void
   getLastQueryRequest?: () => QueryBundleRequest
+  folderId?: string
```

Comment:
Added support for configuring this component with a folderId (might be a projectId), so that the user can add all child files of a container.